css/style.css

```diff
@@ -225,3 +233,53 @@ a {
   line-height: 20px;
   color: #253858;
 }
+
+.contact {
+  background: #6070ff url(../img/footer.svg) 70% no-repeat;
+  box-shadow: 0 -100px 0 #fff;
+  flex-direction: column;
+  padding: 24px;
+  border-radius: 5rem 0 0 0;
+  background-position-x: 138px;
+}
+
+.white {
+  color: #fff;
+}
+
+.contact h2 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: bold;
+  margin-top: 100px;
+  margin-bottom: 10px;
+}
+
+.contact p {
+  font-size: 20px;
+  line-height: 28px;
+  margin-bottom: 40px;
+  text-align: center;
+}
+
+.smooth_border {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.usr_detail {
+  margin-top: 0.5rem;
+  padding: 0.5rem 0.5rem 1rem 0.5rem;
+  width: 100%;
+}
+
+.usr_comment {
+  padding: 0.5rem 0.5rem 0 0.5rem;
+  margin-top: 0.5rem;
+  width: 100%;
+  height: 10rem;
+}
+
+.contact .button {
```

Comment:
- Kindly work on the hover design of your button. (Try to add borders and color type to your button).